### PR TITLE
TIMEFRAME_RATE_LIMIT depends on IR

### PR DESCRIPTION
**Pull request by chiarazampolli**
Hello @catalinristea , 
You can try the high IR with this. If it works, we can merge.
@noferini , @davidrohr , FYI.

### CODE REVIEW

File: DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh [hunk 1]
```diff
@@ -366,6 +366,9 @@ else
   #
   if [[ "0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01" ]]; then
     export TIMEFRAME_RATE_LIMIT=3
+    if (( $(echo "$RUN_IR > 800000" | bc -l) )); then
```
Comment (davidrohr) [inline DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh:369]: What is the content of $RUN_IR? Is is a complicated string, or just an integer?

Comment (chiarazampolli) [reply DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh:369]: The IR is a float that we extract from CCDB at the very beginning, it is here:
https://github.com/AliceO2Group/O2DPG/blob/99361c5fcf9c2db3bf97a57c57c7bfae78aaa1eb/DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh#L149-L155

**Review by davidrohr — APPROVED** [review]
ok, first though one could do the comparison in bash, but apparently it cannot do floating point. Don't think there is a better solution than what you have...

**Comment by catalinristea** [comment]
Ciao,

Thanks, we're giving it a try in grid.

Yours, Catalin.

**Comment by catalinristea** [comment]
Ciao @chiarazampolli,

We need to merge this one, please, as it proved to be a day saver for HIR runs.
You may also want to update  TIMEFRAME_RATE_LIMIT to 6 for lowIR runs.
Thanks!

Yours, Catalin.

**Comment by chiarazampolli** [comment]
Hello @catalinristea , 
I updated the RATE_LIMIT to 6, check that this is what you meant (I thought we used 3 on the grid...).
Chiara

**Comment by catalinristea** [comment]
Ciao @chiarazampolli,

Thanks, but 3 should still be the default. 
6 could help (or not) for very low IR, like ~800 Hz of run 526689 in 22o, and maybe IR < 14 kHz for the runs of 22q. We found more helpful reducing the number of CTFs per job for such runs.

Yours, Catalin.

**Comment by chiarazampolli** [comment]
Ah, sorry, I did not understand what you meant - too many settings recently to change by hand :-) 
Ok, I will put a threshold at 50 kHz.
Chiara

**Comment by chiarazampolli** [comment]
@catalinristea , like this?

**Comment by catalinristea** [comment]
Ciao @chiarazampolli,

Yes, this is what I meant. Thanks, and sorry for the noise.

Yours, Catalin.

**Comment by chiarazampolli** [comment]
Not at all, and great that you checked!
Chiara

**Pull request merged**